app/models/pairing.rb

```diff
@@ -85,6 +85,16 @@ def side_for(player)
   private
 
   def aggregate_scores
```

Comment:
Does this PR also need something to ensure that 0 is recorded in the DB when scores are saved?

Reply:
I'm not sure what you mean. It looks like this is the method that did that already. It's set as a before save action. Do you think that's not enough? Maybe it could do with a better name, or some direct unit tests?

Reply:
Oh!  I thought from the name of the function that this was used in calculating new pairings.  I clearly didn’t read carefully enough. Thanks for setting me straight.

This method is poorly named, for sure, but we can solve that later.

Does this still do the trick if you move this block after the `return unless`?  That seems a bit nicer since it won’t make any changes at all for that case.

Reply:
I've refactored the method, how does it look now?